src/relay/backend/graph_executor_codegen.cc

```diff
@@ -577,10 +748,14 @@ class GraphExecutorCodegen : public backend::MemoizedExprTranslator<std::vector<
   Map<Expr, Array<IntegerArray>> storage_device_map_;
   /*! \brief lowered funcs */
   std::unordered_map<std::string, IRModule> lowered_funcs_;
+  /*! \brief lowered funcs */
+  Map<String, FunctionInfo> function_metadata_;
   /*! \brief name map */
   std::unordered_map<std::string, size_t> name_map_;
   /*! \brief compile engine */
   CompileEngine compile_engine_;
+  /*! \brief main function name */
```

Comment:
can you add a comment to keep in sync with MAIN_FUNC_STR in python/tvm/micro/model_library_format.py? and vice versa there?

Reply:
Done, actually no KMainFuncStr needed as we agreed to use runtime::symbol::tvm_module_main